File: tasks/New 75/task-1/Merge-Strings-Alternately.ts
function mergeAlternately(word1: string, word2: string): string {
    const result: string[] = [];
    const maxLength = Math.max(word1.length, word2.length);

    for (let i = 0; i < maxLength; i++) {
        if (i < word1.length) {
            result.push(word1[i]);
        }
        if (i < word2.length) {
            result.push(word2[i]);
        }
    }

    return result.join('');
}

File: tasks/New 75/task-1/Merge-Strings-Alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        
        result = []

        maxLength = max(len(word1),len(word2) )

        for i in range (maxLength):

            if i < len(word1):
                result.append(word1[i])

            if i < len(word2):
                result.append(word2[i])

        return ''.join(result)